### PR TITLE
doc(example): fix the broken link.

### PR DESCRIPTION
**Pull request by Electronic-Waste**
**What this PR does / why we need it**:

As @yehudit1987 discussed with me in the Slack, there is a missing guide link in the environment setting section : https://github.com/kubeflow/katib/blob/master/examples/v1beta1/kubeflow-pipelines/README.md#multi-user-pipelines-setup 

We decided to fix it to make the guide accessible to new-comers.

cc👀 @yehudit1987 @kubeflow/wg-automl-leads 

**Which issue(s) this PR fixes** _(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)_:
Fixes #

**Checklist:**

- [ ] [Docs](https://www.kubeflow.org/docs/components/katib/) included if any changes are user facing

### CODE REVIEW

File: examples/v1beta1/kubeflow-pipelines/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ You have to install the following Python SDK to run these examples:
 The Notebooks examples run Pipelines in multi-user mode and your Kubeflow Notebook
 must have the appropriate `PodDefault` with the `pipelines.kubeflow.org` audience.
 
-Please follow [this guide](https://www.kubeflow.org/docs/components/pipelines/sdk/connect-api/#multi-user-mode)
+Please follow [this guide](https://www.kubeflow.org/docs/components/pipelines/operator-guides/multi-user/)
```
[the remark this answers is not in the document]
Comment (Electronic-Waste) [reply examples/v1beta1/kubeflow-pipelines/README.md]: It seems that we have multiple authentication methods now. I've updated the link now :)

**Comment by andreyvelich** [comment]
@Electronic-Waste You might need to re-push your commit since we can't re-trigger workflow which is more than one month old.

**Comment by Electronic-Waste** [comment]
@andreyvelich Done!

**Pull request merged**